### PR TITLE
Code r

### PR DESCRIPTION
**Pull request by epatrik107**
javitasok

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java
Comment (MatthewsO3) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:1]: javitasok

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 * (Deliberately contains bugs.)
 */
 public class TorpedoStore {
-
+  private Random random = new Random();
```
Comment (MatthewsO3) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:11]: Use a cryptographically strong random number generator (RNG) like "java.security.SecureRandom" in place of this PRNG.

**Review by MatthewsO3 — CHANGES_REQUESTED** [review]
irjad at

**Comment by epatrik107** [comment]
Use a cryptographically strong random number generator (RNG) like "java.security.SecureRandom" in place of this PRNG.

**Pull request merged**